packages/dom/src/dom.js

```diff
@@ -480,6 +494,10 @@ export function documentHasSelection() {
 		return true;
 	}
 
+	if ( isNumberInput( document.activeElement ) ) {
```

Comment:
Should we add this inside isTextField or should it be a separate function? Or more precisely, do we consider a "number" input as a text field? I might think yes but curious what others think?

Reply:
Initially I extended `isTextField` but later I asked myself the same question and to me it makes more sense to have them separated. I don't consider numbers as text per se. Also curious what others think : )